litr/src/main/java/com/linkedin/android/litr/resample/DownsampleAudioResampler.java

```diff
@@ -0,0 +1,49 @@
+package com.linkedin.android.litr.resample;
+
+import androidx.annotation.NonNull;
+import java.nio.ByteBuffer;
+
+/**
+ * An {@link AudioResampler} that downsamples from a higher sample rate to a lower sample rate.
+ */
+public class DownsampleAudioResampler implements AudioResampler {
```

Comment:
Is this adapted code from a different open source project? In that case we have to keep their copyright notice and add a note that we made modifications (feel free to add your name as author of modifications)